.github/workflows/development-environment.yml

```diff
@@ -15,14 +14,13 @@ jobs:
   dev-environment:
     name: set up
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 35
+    timeout-minutes: 40
     strategy:
       matrix:
-        # macosx-10.15 is Catalina
         # macosx-11.0 is Big Sur, however, it takes long for jobs to get started
         # Using Ubuntu 18 until I figure out this error:
         # -> ImportError: libffi.so.6: cannot open shared object file: No such file or directory
-        os: [ macos-10.15, macos-11.0, ubuntu-18.04 ]
```

Comment:
We don't need Catalina anymore. It seems that it runs on slower machines and it tends to time out.